ta4j-core/src/main/java/org/ta4j/core/BaseBar.java

```diff
@@ -257,12 +299,12 @@ public void addPrice(Num price) {
         closePrice = price;
         if (highPrice == null) {
             highPrice = price;
-        } else if(highPrice.isLessThan(price)) {
```

Comment:
Replace if-else-if with:

```
if (highPrice == null || highPrice.isLessThan(price)) {
highPrice = price;
}
if (lowPrice == null || lowPrice.isGreaterThan(price)) {
lowPrice = price;
}
```

Reply:
Due to existing pull request: https://github.com/ta4j/ta4j/pull/423/files, this comment is no longer valid. It will be fixed in #423

Reply:
oh ok, forget it;)